metron-interface/metron-alerts/src/app/model/alert.ts

```diff
@@ -0,0 +1,44 @@
+export class Alert {
```

Comment:
I agree the name alert might be a misnomer the accurate name should reflect that this class holds all the data in the ES indexes.  Any suggestions?